tests/apps/sequencing_metrics_parser/conftest.py

```diff
@@ -0,0 +1,21 @@
+"""Fixtures for the sequencing metics parser tests."""
+from pathlib import Path
+import pytest
+
+
+@pytest.fixture(name="bcl_convert_demux_metric_file_path")
+def fixture_bcl_convert_demux_metric_file_path() -> Path:
+    """Return a path to a BCLConvert demux metrics file."""
+    return Path("tests/fixtures/apps/sequencing_metrics_parser/bcl_convert_metrics.csv")
```

Comment:
Do not hard code `/`. Let pathlib handle OS specifics. 
```suggestion
    return Path("tests", "fixtures", "apps", "sequencing_metrics_parser", "bcl_convert_metrics.csv")
```